bcda/health/health.go

```diff
@@ -0,0 +1,28 @@
+package health
+
+import (
+	"database/sql"
+	"os"
+
+	"github.com/CMSgov/bcda-app/bcda/client"
+)
+
```

Comment:
Just a thought, and out of scope for this ticket, but should we add a check to ensure we can access the Swagger docs?  Since they are hosted via a different path, maybe we can include it in our health check for BCDA API?  Thoughts?  Maybe we can create a separate ticket for this to be done at a later time.

Reply:
we kinda already hit the swagger docs in new relic, but that is currently only polling every hour. If we need to do it more often than that then I agree with Matt.

Reply:
I was thinking that adding it to the health check would ensure that we fail the deployment if swagger is unreachable at the time we deploy... that way we can roll back to the last stable deployment automatically rather than deploying and letting new relic alerts tell us that swagger is unreachable (after we've already deployed)